tests/unit/httpd_test.cc

```diff
@@ -2295,7 +2295,7 @@ future<> test_client_close_connection(bool chunked) {
                         break;
                     }
                 }
-                out.close().get();
+                out.close().handle_exception([](auto ex){}).get();
```

Comment:
"it may happen that client had already closed its end and there's nothing to flush into" if it happens we will get some concrete error type/message, right? why not to check for it to prevent cases when something else is going wrong and we just ignore it?

Reply:
It's system_error(EPIPE). Added specific handling in next version